commit 42061f3cfd47503830da30330da194cc929aad89
Author: Jie Yu <yujie.jay@gmail.com>
Date:   Mon Feb 3 14:05:57 2020 -0800

    dex: add Dex controller
    
    Add Dex controller for declarative management of Dex configurations.

diff --git a/stable/dex-new/charts/dex-controller-v0.3.0.tgz b/stable/dex-new/charts/dex-controller-v0.3.0.tgz
new file mode 100644
index 0000000..7d3da03
Binary files /dev/null and b/stable/dex-new/charts/dex-controller-v0.3.0.tgz differ
diff --git a/stable/dex-new/requirements.lock b/stable/dex-new/requirements.lock
new file mode 100644
index 0000000..735d0c2
--- /dev/null
+++ b/stable/dex-new/requirements.lock
@@ -0,0 +1,6 @@
+dependencies:
+- name: dex-controller
+  repository: https://mesosphere.github.io/dex-controller/charts
+  version: v0.3.0
+digest: sha256:dfdd2c4b8094118c7bf0a2ec49cec74a95e1f77adcc58b16fa5b0f681340627c
+generated: "2020-02-03T14:04:33.484225-08:00"
diff --git a/stable/dex-new/requirements.yaml b/stable/dex-new/requirements.yaml
new file mode 100644
index 0000000..df278c0
--- /dev/null
+++ b/stable/dex-new/requirements.yaml
@@ -0,0 +1,5 @@
+dependencies:
+- name: dex-controller
+  version: v0.3.0
+  repository: https://mesosphere.github.io/dex-controller/charts
+  condition: dex-controller.enabled, global.dex-controller.enabled
diff --git a/stable/dex-new/values.yaml b/stable/dex-new/values.yaml
index f3279e8..f968ade 100644
--- a/stable/dex-new/values.yaml
+++ b/stable/dex-new/values.yaml
@@ -217,3 +217,6 @@ config:
 
 # frontend:
 #   logoURL: https://example.com/yourlogo.png
+
+dex-controller:
+  enabled: true
